### PR TITLE
Added the option to create a CA cert locally

### PR DESCRIPTION
**Pull request by angelos-p**
We currently use a static CA cert to create the TLS certificates. This PR will give us the option to use an local ansible script to create a new CA locally.

Tested locally on my machine.

### CODE REVIEW

File: k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ The playbook allows you to enable TLS with client authentication and generates
 the certificates for you. You can edit the cfssl configuration by amending the
 json files in  `roles/tls_cert/templates/`
 
+WARNING: The default CA for TLS is statically made as an example. IT SHOULD NOT BE USED FOR PRODUCTION! You need to create your own CA either with your own tools or using the script below.
```
Comment (avestuk) [inline k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md]: Can you wrap lines at 79 char please

**Comment by Arau** [comment]
This solution looks very good and really elegant! Thank you

 I need a bit more time to run it and test it on my own.

File: k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md [hunk 2]
```diff
@@ -31,6 +31,12 @@ The playbook allows you to enable TLS with client authentication and generates
...
+
+To create your own CA, run `sudo ansible-playbook create_ca.yaml`
```
Comment (Arau) [inline k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md]: I'd suggest to remove the `sudo`. You can run the playbook without it. If the Docker user is configured properly in the machine, the user should be able to run docker commands.

**Review by Arau — APPROVED** [review]
This is absolutely fantastic. 

I made a cosmetic change. We usually use 79 characters wrapping for Markdown.

I would change the sudo and merge!

File: k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md (hunk 1 above)
Comment (Arau) [reply k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md]: I pushed with that change

**Comment by avestuk** [comment]
LGTM, merge at your leisure.

**Pull request merged**